example/server.js

```diff
@@ -91,6 +92,8 @@ setInterval(() => {
 
 server.get('/metrics', async (req, res) => {
 	try {
+		// register.contentType = Registry.OPENMETRICS_CONTENT_TYPE;
+		register.contentType = Registry.PROMETHEUS_CONTENT_TYPE;
```

Comment:
should go through `setContentType` I guess?

Reply:
Thanks, yeah there is a lot of inconsistency in how that flag is set, I'll iron them out.